Compatibility issue with .vimrc from spf13

Something in the .vimrc provided here https://github.com/spf13/spf13-vim/blob/3.0/.vimrc seems to prevent processing-vim from working. 

Syntax is not colored and ```:make``` just returns:

```make: *** No targets specified and no makefile found.  Stop.``` 

Before I installed spf13, I had Processing-vim installed with Vundle which worked fine.

--- a/ftdetect/processing.vim
+++ b/ftdetect/processing.vim
@@ -1,2 +1,2 @@
-
-au BufRead,BufNewFile *.pde setf processing
+" Processing
+au BufRead,BufNewFile *.pde		set filetype=processing

